blocks/rich-text/index.js

```diff
@@ -558,6 +559,44 @@ export class RichText extends Component {
 		if ( keyCode === BACKSPACE ) {
 			this.onChange();
 		}
+
+		// `scrollToCaret` is called on `nodechange`, whereas calling it on
+		// `keyup` *when* moving to a new RichText element results in incorrect
+		// scrolling. Though the following allows false positives, it results
+		// in much smoother scrolling.
+		if ( keyCode !== BACKSPACE && keyCode !== ENTER ) {
+			this.scrollToCaret();
```

Comment:
Do we really want to run this stuff on every keystroke, down the line: https://gist.github.com/paulirish/5d52fb081b3570c81e3a#range.

Reply:
Or maybe this is fine if there are no layout changes.

Reply:
_That's_ the list I was looking for the other week!

Reply:
Yeah but the layout is not invalidated... so this seems fine. I don't see any issues atm.